source/particle/property/velocity_gradient.cc

```diff
@@ -0,0 +1,98 @@
+/*
+  Copyright (C) 2015 - 2025 by the authors of the ASPECT code.
+ This file is part of ASPECT.
+ ASPECT is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2, or (at your option)
+ any later version.
+ ASPECT is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+ You should have received a copy of the GNU General Public License
+ along with ASPECT; see the file LICENSE.  If not see
+ <http://www.gnu.org/licenses/>.
+ */
+
+#include <aspect/particle/property/velocity_gradient.h>
+#include <aspect/particle/property/interface.h>
+#include <aspect/simulator_access.h>
+
+namespace aspect
+{
+  namespace Particle
+  {
+    namespace Property
+    {
+      template <int dim>
+      void
+      VelocityGradient<dim>::initialize_one_particle_property(const Point<dim> &,
+                                                              std::vector<double> &data) const
+      {
+        const static Tensor<2,dim> identity = unit_symmetric_tensor<dim>();
+        for (unsigned int i = 0; i < Tensor<2,dim>::n_independent_components ; ++i)
+          data.push_back(identity[Tensor<2,dim>::unrolled_to_component_indices(i)]);
+
+      }
+
```

Comment:
three empty lines between function implementations in the .cc file, here and below.